.build/mainnet.json

```diff
@@ -0,0 +1,4 @@
+{
+  "AnchoredPriceView": "0xDECee5C06B8CC6c4D85dfd8518C246fd1497b2AE",
```

Comment:
is this still up-to-date? should AnchoredView  `0x4345d279E3aeA247Ad7865cC95BeE3088eaBc433` be added?

Reply:
yes, forgot topush, thanks